frameworks/hdfs/src/main/dist/hdfs_svc.yml

```diff
@@ -0,0 +1,170 @@
+name: {{SERVICE_NAME}}
+principal: {{SERVICE_PRINCIPAL}}
+zookeeper: master.mesos:2181
+api-port: {{PORT0}}
+pods:
+  journal:
+    count: 3
+    tasks:
+      node:
+        goal: RUNNING
+        cmd: "env && ./resolve-dns.sh && ./hadoop-2.6.0-cdh5.7.1/bin/hdfs journalnode"
```

Comment:
To be clear, this could be something like:
- an envvar thats exposed in a config setting buried somewhere
- or conversely it'd likely be possible to just run something like `./hadoop-*/bin/hdfs` here and have it still work